.nvmrc

```diff
@@ -1 +1 @@
-lts/erbium
+12.13.0
```

Comment:
pourquoi?

Reply:
Oh - github actions was failing because it was using the wrong version. Will revert if it's not needed 👍